state-chain/pallets/cf-ingress-egress/src/lib.rs

```diff
@@ -1310,6 +1326,50 @@ pub mod pallet {
 
 			Ok(())
 		}
+
+		#[pallet::call_index(14)]
+		#[pallet::weight(T::WeightInfo::open_private_channel())]
+		pub fn open_private_channel(origin: OriginFor<T>) -> DispatchResult {
+			let broker_id = T::AccountRoleRegistry::ensure_broker(origin)?;
+
+			ensure!(
+				!BrokerPrivateChannels::<T, I>::contains_key(&broker_id),
```

Comment:
Should we also restrict this to be just for Bitcoin - it's not useful for any other chains right? probably more importantly is that not restricting it to Bitcoin would be confusing when reading the code, since it's an instantiable pallet, we'd assume it's for all instances.

Reply:
Added the check.